src/shared/utils/helpers.ts

```diff
@@ -19,3 +19,15 @@ export const stringToCamelCase = (str: string): string =>
   str.toLowerCase().replace(/[^a-zA-Z0-9]+(.)/g, (_, chr) => chr.toUpperCase())
 
 export const capitalize = (str: string): string => str[0].toUpperCase() + str.slice(1)
+
```

Comment:
we should rename `helpers.ts` to `string-utils.ts`, as all the functions perform string manipulation. I'll do that a bit later

Reply:
ok, sounds good